### ISSUE TITLE
Provide an invoke API that returns details about the invocation like logs

### ISSUE THREAD

**Issue by asavaritayal** [Azure/azure-functions-host#2611]
\cc @ahmelsayed @pragnagopa 

The same fix should apply to bring-your-own-files in consumption as well.

**Comment by pragnagopa** [comment]
For Linux App, Logs on portal do not work as there is no Kudu endpoint. Need to investigate if [App Insights live stream ](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-live-stream) is a work around

**Comment by ahmelsayed** [comment]
I'm removing the `platform:linux` label and renaming the issue. 

I added log polling through vfs to the UI. It's not ideal, but it unblocks linux portal development. Ideally we will get an invoke api that returns the logs as well.

**Comment by brettsam** [comment]
There should be another issue tracking this already -- I will find it.

**Comment by brettsam** [comment]
This looks like the original issue; https://github.com/Azure/azure-functions-host/issues/1329
